OMEdit/OMEditLIB/FlatModelica/Expression.cpp

```diff
@@ -1629,7 +1629,7 @@ namespace FlatModelica
 
   void Unary::print(std::ostream &os) const
   {
-    os << _op << _e;
+    os << _op << " " << _e;
```

Comment:
I guess this is to fix the output of `not`, but then it becomes wrong for `-` instead which shouldn't have a space between. I think adding a space here if the operator is `not` is probably the best solution.

Reply:
Yes to fix the output of `not`. I believe we also need space for `or` and `and`. I am not sure which operators require space. Can you please fix this part?

Reply:
Binary operators are always printed with spaces, the only issue is with the unary operators `not` and `-` where `not` requires a space while `-` should not have a space. So e.g.:
```
os << _op;

if (_op.type() == Operator::Not) {
  os << " ";
}

os << _e;
```